pages/common/pageres.md

```diff
@@ -0,0 +1,29 @@
+# pageres
+
+> Capture screenshots of websites in various resolutions.
+> More information: <https://github.com/sindresorhus/pageres-cli>.
+> More information on API: <https://github.com/sindresorhus/pageres>.
```

Comment:
The API link is reachable from the CLI page also. I would suggest to remove this and keep just one.